setup.json

```
@@ -14,10 +14,10 @@
     ],
     "extras_require": {
         "pre-commit": [
-            "pre-commit==1.17.0",
-            "yapf==0.28.0",
-            "prospector==1.1.7",
-      	    "pylint==2.3.1"
+		"pre-commit==1.17.0",
```

Newest version is 2.6.0. For the major version 1, the newest version is 1.21.0.
```suggestion
		"pre-commit==2.6.0",
```